testsuite/MDAnalysisTests/topology/base.py

```diff
@@ -36,14 +36,18 @@ class ParserBase(object):
     expected_attrs = []
     guessed_attrs = []
 
+    @pytest.fixture
+    def filename(self):
```

Comment:
I still think it's ugly that we have to redefine this fixture in child classes to have more than one filename.

Reply:
I played around a lot.  The problem is I can't define a fixture using the `params` kwargs in the baseclass with a variable name `ref_filename` and then fill that variable name in the childclasses with meaningful values.

Reply:
Ok thanks then, maybe this is the cleanest solution